Libraries/Components/RefreshControl/RefreshControl.js

```diff
@@ -137,11 +153,11 @@ const RefreshControl = React.createClass({
   },
 
   _onRefresh() {
+    this._lastNativeRefreshing = true;
+
     this.props.onRefresh && this.props.onRefresh();
 
-    if (this._nativeRef) {
-      this._nativeRef.setNativeProps({refreshing: this.props.refreshing});
-    }
+    this.forceUpdate();
```

Comment:
Instead of re-rendering this component, don't you want to just run the same code as componentnDidUpdate?